src/app/projects/[project_id]/browse-files/page.tsx

```diff
@@ -213,6 +272,53 @@ export default function BrowseFiles({ params }: { params: { project_id: string }
         isLoadingFileContent={isLoadingFileContent}
         fileContent={fileContent}
       />
+      {/* Suggestions Panel */}
```

Comment:
we could prob compentize this (but let's leave it for next week lol)

Reply:
Yeah agreed we should throw all 3 sections into resizable